pkg/metadata/inventories/inventories.go

```diff
@@ -216,4 +223,11 @@ func StartMetadataUpdatedGoroutine(sc schedulerInterface, minSendInterval time.D
 func InitializeData() {
 	SetAgentMetadata(AgentVersion, version.AgentVersion)
 	SetAgentMetadata(AgentFlavor, flavor.GetFlavor())
+
+	SetAgentMetadata(AgentSecurityEnabled, config.Datadog.GetString("runtime_security_config.enabled"))
+	SetAgentMetadata(AgentProcessEnabled, config.Datadog.GetString("process_config.enabled"))
+	SetAgentMetadata(AgentNetworksEnabled, config.Datadog.GetString("system_probe_config.enabled"))
```

Comment:
NPM and system-probe are no longer explicitly linked since there are multiple modules within system-probe now. If you need the ability to read this from `config.Datadog`, we should set `network_config.enabled` [here](https://github.com/DataDog/datadog-agent/blob/3437f30c6b6152b78e7389f81c4dd893b5479724/cmd/system-probe/config/config.go#L178) and use `network_config.enabled` here.

Reply:
Thanks for the review, I used `network_config.enabled` instead. But this the inventory payload only sent from the agent right now, updating cmd/system-probe would not help.
Instead we send `false` is the setting is not set or set to empty string.

Reply:
The agent calls `common.SetupSystemProbeConfig` which uses `cmd/system-probe/config` to merge in the values from `system-probe.yaml`. See https://github.com/DataDog/datadog-agent/blob/main/cmd/agent/common/system_probe_config_helper.go#L16